src/main/java/com/cinema/model/Order.java

```diff
@@ -0,0 +1,34 @@
+package com.cinema.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@RequiredArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @NonNull
+    @OneToMany
+    private List<Ticket> tickets;
+    @NonNull
+    private LocalDateTime orderDate;
+    @NonNull
+    @ManyToOne
```

Comment:
<img width="620" alt="Screenshot 2020-10-12 at 13 08 48" src="https://user-images.githubusercontent.com/68664175/95734395-14de4d80-0c8c-11eb-8d5b-c10b9595ada6.png">

Reply:
fixed

Reply:
Sorry for confusing
This is redundant
It was about tickets, movies etc, not users 
My apologies